modules/ROOT/pages/propagate-metadata-studio.adoc

```diff
@@ -0,0 +1,161 @@
+= Propagate Metadata to Other Flows
+
+Studio enables you to propagate your flow's metadata to your referenced flows within your application.
```

Comment:
Is this the same? Just trying to limit the use of "you(r)" in the sentence...lol.
```suggestion
Studio enables you to propagate your flow's metadata to other flows referenced within your application.
```